loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -474,6 +474,76 @@ module BABYLON.GLTF2 {
             }
         }
 
+        /**
+         * Converts a data bufferview into a Float4 Texture Coordinate Array, based on the accessor component type
+         * @param {ArrayBufferView} data 
+         * @param {IGLTFAccessor} accessor 
+         */
+        private _convertToFloat4TextureCoordArray(data: ArrayBufferView, accessor: IGLTFAccessor): Float32Array {
+            const buffer = <Float32Array>data;

```

Comment:
buffer is not always a Float32Array. not sure what the consequences of that is. should probably cast to the correct type based on componentType.